docs/specs/perception_apollo_3.0.md

```diff
@@ -3,7 +3,8 @@ Apollo 3.0
 June 27, 2018
 
 ## Introduction
-Apollo 3.0 aims for L2 autonomous driving with low cost sensors. The autonomous vehicle will stay in the lane and keep a distance with the closest in-path vehicle (CIPV) using a single front-facing camera and a frontal radar. Apollo 3.0 is best suited for a closed venue setting and driving should be restricted to low speed. 
+
+Apollo 3.0 supports L2 autonomous driving capabilities with low cost sensors. The autonomous vehicle will stay in the lane and keep a distance with the closest in-path vehicle (CIPV) using a single front-facing camera and a frontal radar. Apollo 3.0 is best suited for a closed venue setting and driving should be restricted to low speed. 
```

Comment:
This statement is missleading. Apollo 3.0 covers both local and freeway.

Reply:
Recommend using the introduction from https://medium.com/apollo-auto/apollo-3-0-entering-the-new-era-of-autonomous-driving-d781bc769cef

--- Apollo 3.0 introduced a production level solution for the low-cost, closed venue driving scenario that is used as the foundation for comercialized products.

Reply:
Yup agreed! Changed made as requested.